### PR TITLE
KIT-28 Create Makefile & disable deprecated linters

### PR DESCRIPTION
**Pull request by platsko**
Create Makefile with targets:
1. pre-push & check-commit
1.1. go get -d ./...
1.2. go test ./...
1.3. golangci-lint run --fix ./...

2. go-mod tidy & download
2.1. go mod tidy -v
2.2. go mod download -x

3. protoc upgrade & generate
3.1. go get -u github.com/golang/protobuf/protoc-gen-go@latest
3.2. go get -u google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
3.2. execute command: protoc ...

Update golangci-lint config file to disable deprecated linter(s).

Update README.md - add info of usage make command on WINDOWS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,43 @@
 
 [![Go](https://github.com/evenlab/go-kit/actions/workflows/go.yml/badge.svg)](https://github.com/evenlab/go-kit/actions/workflows/go.yml)
 
-There are helpful lightweight and tested functionality for reuse in other projects
+### Golang KIT of helpful lightweight and tested functionality for reuse in other projects
+
+# `Makefile` Usage on Windows
```
Comment (Vad1mAlekseev) [inline README.md:7]: Semantically, headers (starts with '#') shouldn't contain sections with code (starts with ` or ```)
I suggest removing them and rephrasing with: Usage GNU Make on Windows;  Setup Choco package manager;

File: README.md [hunk 2]
```diff
@@ -2,4 +2,43 @@
...
+
+## `choco` Setup Chocolatey CLI
+
+### Requirements:
+* Windows 7+ / Windows Server 2003+
+* PowerShell v2+ (Not PowerShell Core yet though)(minimum is v3 for install from this website due to TLS 1.2 requirement)
+* .NET Framework 4+ (the installation will attempt to install .NET 4.0 if you do not have it installed)(minimum is 4.5 for install from this website due to TLS 1.2 requirement)
+> That's it! All you need is choco.exe (that you get from the installation scripts) and you are good to go! No Visual Studio required.
+
+### Installing Chocolatey
```
Comment (Vad1mAlekseev) [inline README.md:17]: Maybe we should be redirecting to the Choco PM installation page? If the installation method changes, we will have to edit the README (which is not good (write README according to SOLID 😅))

File: README.md (hunk 1 above)
Comment (platsko) [reply README.md:7]: I'm not sure that ` is for blocks of code, but to highlight some text I guess...

**Review by Vad1mAlekseev — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (platsko) [reply README.md:19]: I believe there is not much to change there.

**Pull request merged**